azimuth_caas_operator/utils/ansible_runner.py

```diff
@@ -173,19 +175,6 @@ def get_job(
         fsGroup: 1000
       restartPolicy: Never
       initContainers:
-      - image: "{image}"
```

Comment:
I think we still need this to keep all the existing appliances working, and if you override that value, it will just get ignored anyway?

Reply:
Yes, I'm just trying to get it working for Slurm first, then I can work out the best way of making it backwards compatible!